libs/types/apollo.config.js

```diff
@@ -2,7 +2,8 @@ module.exports = {
   client: {
     service: {
       name: 'vega',
-      url: process.env.NX_VEGA_URL,
+      url:
+        process.env.NX_VEGA_URL || 'https://api.n11.testnet.vega.xyz/graphql',
```

Comment:
are you happy with this @mattrussell36 ?

Reply:
yup